backend/buildeR.BLL/Services/UserService.cs

```diff
@@ -121,5 +121,32 @@ public async Task<bool> ValidateUsername(ValidateUserDTO user)
                 return await _context.Users.AnyAsync(x => x.Username.ToLower() == user.Username.ToLower());
             }
         }
+
+        public async Task<UserDTO> LinkProvider(LinkProviderDTO userLink)
+        {
+            var isUserExist = await _context.Users.AnyAsync(u => u.Id == userLink.UserId);
+
+            if (isUserExist)
+            {
+                var userSN = new NewUserSocialNetworkDTO()
+                {
+                    UId = userLink.UId,
+                    SocialNetworkId = (int)userLink.ProviderId + 1,
```

Comment:
What are you doing here?

Reply:
Maybe, I had to change enum (starting from 1), now it starts from 0, that's why